dashboard/modules/log/tests/test_log.py

```diff
@@ -149,5 +150,53 @@ def test_log_proxy(ray_start_with_dashboard):
                 raise Exception(f"Timed out while testing, {ex_stack}")
 
 
+def test_logs_experimental_index(ray_start_with_dashboard):
+    assert wait_until_server_available(ray_start_with_dashboard["webui_url"]) is True
+    webui_url = ray_start_with_dashboard["webui_url"]
+    webui_url = format_web_url(webui_url)
+    node_id = ray_start_with_dashboard["node_id"]
+
+    timeout_seconds = 10
+    start_time = time.time()
+    last_ex = None
+    while True:
+        time.sleep(1)
+        try:
+            response = requests.get(webui_url + "/api/experimental/logs/index")
```

Comment:
The test doesn't cover following scenarios

- filtering of list logs
- Actual streaming logs
- Streaming logs when files are actively updated
- Just regular log print

Reply:
added test for streaming (w/ live update), tailing, actor id, and logs list filtering